src/CfgEditor/CfgEditorPanel.ts

```diff
@@ -246,6 +283,18 @@ export class CfgEditorPanel implements vscode.CustomTextEditorProvider {
           x.dispose();
         }
       }
+
+      if (this._activeWebviewPanel === webviewPanel) {
+        // there are race conditions with visibility
+        // another webviewPanel becomes visible while this one is disposed
+        // so 'activeCfgIsVisible' will not be set to false
+        vscode.commands.executeCommand(
+          "setContext",
+          "one.CfgEditor:activeCfgIsVisible",
+          false
+        );
+      }
```

Comment:
By using the given context below, you don't need to be bothered to implement this custom context.

> "when": "activeCustomEditorId == one.editor.cfg"

Reply:
Ahh... Ok. Thank you. I'll try!